lib/ansible/modules/cloud/smartos/vmadm.py

```diff
@@ -78,6 +78,11 @@
     required: false
     description:
       - Domain value for C(/etc/hosts).
+  docker:
+    required: false
+    description:
+      - Enable Docker mode
```

Comment:
Is it obvious what this means, could any extra information be given?
also missing full stop.

Reply:
the whole docker mode isn't exactly obvious to use, but I changed the description